assets/helpers/internationalisation/countryGroup.js

```diff
@@ -154,7 +160,15 @@ function detect(): CountryGroupId {
   return fromPath() || fromQueryParameter() || fromCookie() || fromGeolocation() || 'GBPCountries';
 }
 
+function stringToCountryGroup(cg: string): CountryGroupId {
```

Comment:
maybe a better name for `cg` like `tentativeCountryGroupId`

Reply:
I would say that since here the type is string probably `countryGroupId` is a better choice than `cgId`.